intl/msg_hash_us.h

```diff
@@ -6793,43 +6793,43 @@ MSG_HASH(
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_KANNADA,
-   "Kannada"
+   "Kannada - ಕನ್ನಡ (Restart Required)"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_KOREAN,
    "Korean - 한국어 (Restart Required)"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_LATIN,
-   "Latin"
+   "Latin - Latine"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_LATVIAN,
-   "Latvian"
+   "Latvian - Latviešu Valoda"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_LITHUANIAN,
-   "Lithuanian"
+   "Lithuanian - Lietuvis"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_MACEDONIAN,
-   "Macedonian"
+   "Macedonian - Македонски"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_MALAY,
-   "Malay"
+   "Malay - Bahasa Melayu"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_MALTESE,
-   "Maltese"
+   "Maltese - Malti"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_NORWEGIAN,
-   "Norwegian"
+   "Norwegian - Norveġiż"
```

Comment:
nope, that should be Norsk.

Reply:
Fixed! Thanks a lot, I tried to translate every language name correctly, I will double-check again on Wikipedia to make sure it's the correct language name (and not the people's name or something)…

Reply:
Wikipedia translates the following languages differently from us:

| English | Ours | Wikipedia |
| --- | --- | --- |
| Albanian | Shqiptar | Shqip |
| Arabic | عربى | العربية |
| Azerbaijani | Azərbaycan | Azərbaycanca |
| Basque | Euskal | Euskara |
| Bulgarian | български | Български |
| Chinese (Simplified)<br>Chinese (Traditional) | 繁體中文<br>简体中文 | 中文 |
| Estonian | Eestlane | Eesti |
| Filipino | Pilipino | don't know, Tagalog maybe? |
| Haitian Creole | Kreyòl Ayisyen | Kreyòl ayisyen (case difference) |
| Hebrew | עִברִית | עברית |
| Hungarian | Magyar nyelv | Magyar |
| Icelandic | Íslensku | Íslenska |
| Latin | Latine | Latina |
| Latvian | Latviešu Valoda | Latviešu |
| Lithuanian | Lietuvis | Lietuvių |
| Norwegian | Norsk | Norsk bokmål<br>Norsk nynorsk |
| Portuguese | Português (Portugal)<br>Português (Brasil) | Português |
| Serbian | Српски | Српски / srpski |
| Slovak | Slovenský | Slovenská |
| Slovenian | Slovenščina | Slovenska (I think, their names are confusingly similar) |
| Tamil | தமிழ் மொழி | தமிழ் |
| Ukrainian | українська мова | Українська |
| Yiddish | יידיש | ייִדיש |

I'm not saying we're wrong on all of those - splitting Norwegian, Chinese and Portuguese differently doesn't make either of us wrong, and our Cyrillic-only Serbian isn't wrong either.

However, most of them do look like bugs. For example, Shqiptar is Albanian people, "Valoda" is Latvian for 'language', and we're missing a couple of dots on Hebrew and Yiddish. And I don't know what Arabic is doing.

(I didn't check which of those divergences are added by this PR, and which are in master already. It's not an interesting distinction anyways, both types should be fixed.)

Reply:
Thanks a lot!

I just corrected all the language names I got wrong by using [this website](https://omniglot.com/language/names.htm), but I double-checked and it seems it got Croatian wrong (it says Hravtski on the website but Hrvatski on Wikipedia…), and for Arabic, I chose the modern standard name…

For Filipino, Wikipedia seems to say it's translated Wikang Filipino, but is also known as Pilipino… So am I right to translate it to Wikang Filipino? (It seems to imply that Pilipino is an english thing if I'm not mistaken…) Link: https://en.wikipedia.org/wiki/Filipino_language

I didn't change the simplified and traditional variants of Chinese because it would have been caught by @guoyunhe if they were wrong…

Reply:
Wikang means Language, so I'm gonna vote let Filipino translate to itself, like Afrikaans and Esperanto (and English maybe?).

Dili (Azerbaijani), Keel (Estonian), ენა (Georgian), Nyelv (Hungarian), Lingua (Latin), Valoda (Latvian), Kalba (Lithuanian), Limba (Romanian), язык (Russian), and мова (Ukrainian) also mean Language.

As I said, the Chineses are correct as is, they're just Wikipedia only having one Chinese edition. No opinion on Arabic and Croatian, let's use whatever's there and see if anyone complains.

Reply:
Actually Wikang Filipino can be a case like Indonesian, where it's not correct to say Bahasa (which means Language in Indonesian – and Malay BTW) but it's not correct neither to say Indonesia for obvious reasons, so you HAVE to say Bahasa Indonesia to be correct…

Reply:
Okay, if you're more confident than me, let's go with yours.

I also vote leave all existing translations as is, i.e. undo changes to Arabic, Russian and Slovak. If they're wrong, let people who actually know those languages point that out; if they're right, we don't want to break it.

Reply:
Actually, Wikipedia translates Russian to русский язык https://en.wikipedia.org/wiki/Russian_language
Arabic to اَلْعَرَبِيَّةُ‎ (I just corrected my version which was missing some accents) https://en.wikipedia.org/wiki/Arabic
and Slovak to Slovenčina or Slovenský Jazyk (not just Slovenský like I did before) https://en.wikipedia.org/wiki/Slovak_language